src/main/kotlin/co/yappuworld/team/client/application/AdminTeamService.kt

```
@@ -0,0 +1,304 @@
+    }
+
+    @Transactional
+    fun deleteTeam(teamId: UUID) {
+        val team = teamFindService.findTeam(teamId)
+
+        val members = teamMemberFindService.findMembers(team)
+        if (members.isNotEmpty()) {
+            val memberIds = members.map { it.id }
+            try {
+                teamMemberCommandService.deleteAll(memberIds)
+            } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)